scaper/core.py

```diff
@@ -1768,6 +1793,7 @@ def _generate_audio(self, audio_path, ann, reverb=None,
         # List for storing all generated audio (one array for every event)
         soundscape_audio = None
         event_audio_list = []
+        scale_factor = None
```

Comment:
Instead of `None`, you could say the `scale_factor` defaults to `1.0`, which might make the logic in the JAMS annotation easier to handle.

Reply:
done